src/aws-manager.js

```diff
@@ -349,7 +351,7 @@ class AwsManager {
             },
           ],
         });
-        await delayer(1000)();
+        await delayer(describeSpotInstanceDelay)();
```

Comment:
These are describing spot requests, not spot instances.  It's probably minor, but it might be confusing to use this name